kiwi/volume_manager/btrfs.py

```diff
@@ -439,6 +443,17 @@ def set_property_readonly_root(self):
                 ['btrfs', 'property', 'set', sync_target, 'ro', 'true']
             )
 
+    def _apply_quota(self, volume_path: str, attributes: List[str]):
+        for attribute in attributes:
+            if attribute.startswith('quota='):
+                quota = attribute.split('=')[1]
+                Command.run(
+                    ['btrfs', 'quota', 'enable', volume_path]
+                )
+                Command.run(
+                    ['btrfs', 'qgroup', 'limit', quota, volume_path]
+                )
```

Comment:
We should not use qgroups. We should use simple quotas instead. https://lwn.net/Articles/944371/

Reply:
Now I'm having a problem. In the issue description the reported suggested to use qgroups. I was also reading about qgroups and the simple groups and I have no idea what values one over the other.

```
However, due to btrfs's high level decision to tradeoff snapshot
performance against ref-counting performance, qgroups suffer from
non-trivial performance issues that make them unattractive in certain
workloads.
```

Is that the justification to move ?

I'm fine with it, but wanted to double check

Reply:
Yeah, @boryas added it because of this.

Reply:
And what's the commandline API to make use of squota ? It's somehow quite hard to find commandline docu. I was reading this one: https://btrfs.readthedocs.io/en/latest/Qgroups.html#simple-quotas-squota

Reply:
Just use `btrfs quota enable --simple` instead of `btrfs quota enable`, I believe.

Reply:
Thanks, that helped to search it 

```
==API==
Squotas reuses the api of qgroups. The only difference is that when you
enable quotas via `btrfs quota enable`, you pass the `--simple` flag.
Squotas will always report exclusive == shared for each qgroup. Squotas
deal with extent_item/metadata_item sizes and thus do not do anything
special with compression. Squotas also introduce auto inheritance for
nested subvols. The API is documented more fully in the documentation
patches in btrfs-progs.
```

I don't think all of our supported distros have it

Reply:
I applied that and will check if the integration test will build

Reply:
It was added in btrfs-progs v6.5.2, which was a year ago.

Reply:
```
iosc ls SUSE:SLE-15-SP6:Update/btrfsprogs
boot-btrfs.sh
btrfs-progs-v6.5.1.tar.gz
```

Reply:
anyway let's go this way and react when there are complains